cwms-data-api/src/main/java/cwms/cda/data/dto/forecast/ForecastFile.java

```diff
@@ -1,23 +1,26 @@
 package cwms.cda.data.dto.forecast;
 
-import javax.xml.bind.annotation.XmlAccessType;
-import javax.xml.bind.annotation.XmlAccessorType;
-import javax.xml.bind.annotation.XmlAttribute;
-import javax.xml.bind.annotation.XmlRootElement;
+import io.swagger.v3.oas.annotations.media.Schema;
 
-@XmlRootElement(name = "alias")
+import javax.xml.bind.annotation.*;
+
+@XmlRootElement(name = "forecast-file")
 @XmlAccessorType(XmlAccessType.FIELD)
 public class ForecastFile {
 
-  @XmlAttribute
-  private Integer fcstInstCode;
+  @Schema(description = "Forecast Instance ID")
+  @XmlElement(name = "instance-id")
+  String instanceId;
 
-  @XmlAttribute
-  private Integer blobCode;
+  @Schema(description = "File's Blob ID")
+  @XmlElement(name = "blob-id")

```

Comment:
We're just embedding the "blob" directly or maybe the "file" objects I'm creating for the text timeseries. not entirely sure but this should at at least be "byte forecastData[]" and not a blob Id reference.

Reply:
Actually wait shouldn't the forecast data here be per instance?

Reply:
Right this is linked to the instance, so should the blob and file name/description be embedded in the ForecastInstance DTO?

Reply:
Yeah.